MapboxCoreNavigation/RouteController.swift

```diff
@@ -172,16 +172,13 @@ open class RouteController: NSObject {
         
         let status = navigator.getStatusForTimestamp(location.timestamp)
         
-        let currentStepProgress = routeProgress.currentLegProgress.currentStepProgress
-        currentStepProgress.intersectionIndex = Int(status.intersectionIndex)
-        
         // Notify observers if the step’s remaining distance has changed.
         update(progress: routeProgress, with: CLLocation(status.location), rawLocation: location)
         
         let willReroute = !userIsOnRoute(location) && delegate?.router?(self, shouldRerouteFrom: location)
                           ?? DefaultBehavior.shouldRerouteFromLocation
         
-        updateRouteStepProgress(status: status)
+        updateIndexes(status: status)
```

Comment:
nit: plural of `index` is `indices`

Reply:
both are valid pluralizations, `indexes` is even being used in Foundation, e.g. `IndexPath.init(indexes:)`.

Reply:
That's fine. Realized it's accepted in American and Canadian English 😸